example/src/AsyncExample.js

```diff
@@ -122,6 +122,14 @@ export default class AsyncExample extends React.Component {
             name="RNLocalize.uses24HourClock()"
             value={RNLocalize.uses24HourClock()}
           />
+          <Line
```

Comment:
Maybe we should display the value only if the functions are available on the platform?

```tsx
{Platform.OS === "android" && <>
  <Line
    name="RNLocalize.isAutoDateAndTime()"
    value={RNLocalize.isAutoDateAndTime()}
  />
  <Line
    name="RNLocalize.isAutoTimeZone()"
    value={RNLocalize.isAutoTimeZone()}
  />
</>}
```

Reply:
related to the above philosophy thinking on platform-specific APIs and defaults etc. I'll wait till you think on that in general.